moveit_ros/planning/CMakeLists.txt

```diff
@@ -79,9 +77,6 @@ include_directories(SYSTEM
                     ${EIGEN3_INCLUDE_DIRS}
 )
 
-link_directories(${Boost_LIBRARY_DIRS})
-link_directories(${catkin_LIBRARY_DIRS})
```

Comment:
catkin auto links this, correct?

Reply:
All find_package'd stuff usually works correctly. However, pkg-config doesn't provide full library paths.